src/export/csv/index.ts

```diff
@@ -0,0 +1,101 @@
+import { GraphQLClient } from "graphql-request";
+import omit from "lodash.omit";
+import { getFlowName, getLatestFlowGraph } from "../../flow";
+import { getSessionById } from "../../session";
+import { getBOPSParams } from "../bops/index";
+import { QuestionAndResponses, Response } from "../bops/model";
+import { CSVData } from "./model";
+
+export async function generateCSVData(
+  client: GraphQLClient,
+  sessionId: string
+): Promise<CSVData> {
+  const session = await getSessionById(client, sessionId);
```

Comment:
This may be in another PR, but I believe using `generateCSVData()` will only work via the `_admin` client due to missing permissions - we won't be able to use this as-is in the frontend.

We can address this in a number of ways but it's probably something pretty fundamental to how this library works that merits a discussion. I've added it to the dev call agenda 👍